src/watchmaker/workers/salt.py

```diff
@@ -5,6 +5,7 @@
 
 import ast
 import codecs
+import distutils
```

Comment:
Can we use `shutil.copytree`, rather than introducing a new import?

Reply:
`shutil.copytree` is picky: 

> The destination directory, named by _dst_, must not already exist; it will be created.

Here `extract_dir` exists when `shutil.copytree` would be used. `dir_util.copy_tree` does not require that the destination directory not exist. If _dst_/`extract_dir` is not important, I can remove it and recreate with `shutil.copytree`.

Reply:
I think we _must_ remove the directory. That we're not may be a latent bug. Would need to trace through the setup of the salt srv directory structure to be sure.

Reply:
Will do. 👍